command.c

```diff
@@ -2513,9 +2513,7 @@ TODO: Add a setting for these tweaks */
          ui_companion_event_command(cmd);
          break;
       case CMD_EVENT_SHADERS_APPLY_CHANGES:
-#ifdef HAVE_MENU
```

Comment:
Why did you get rid of this HAVE_MENU ifdef? We should allow for the possibility of being able to compile without the menu.

Reply:
The function has an internal HAVE_MENU ifdef, so I thought this was some kind of bug.
I undid this locally however because I don't understand it well enough, so this won't be in the finished pull request.